tools/src/main/scala/scala/scalanative/linker/ClassPath.scala

```diff
@@ -55,9 +56,9 @@ object ClassPath {
 
     lazy val classesWithEntryPoints: Iterable[Global.Top] = {
       files.filter {
-        case (top, file) =>
-          val buffer = directory.read(file, len = 1)
-          buffer.get() != 0
+        case (_, file) =>
+          val buffer = directory.read(file, len = 16)
```

Comment:
This 16 is magic, as seen from here. It should be a `val` in `NirPrelude` (and it probably be should 13, not 16).